surrogates/common_utils/doc_string.py

```diff
@@ -0,0 +1,141 @@
+##==============================================================================
+## BHPTNRSurrogate module
+## Description : docs for different surrogate model wrappers
+## Author : Tousif Islam, Nov 2022 [tislam@umassd.edu / tousifislam24@gmail.com]
+##==============================================================================
+
+import inspect
+
+def copy_doc(base_func, model_func):
+    """
+    function to copy doc string from the base function and the model function
+    and join them to create docstring fot the target function
+    """
+    def wrapper(target_func):
+        target_func.__doc__ = base_func.__doc__ + model_func.__doc__
+        return target_func
+    return wrapper
+    
+    
+def generic_doc_for_models() -> None:
+    """
+    ## -------------------------------------------------------------------------- ##
+    ## general overview
+    ## -------------------------------------------------------------------------- ##
+    
+    Description : warpper to generate BHPT surrogate waveforms
+    
+    Input
+    =====
+    q:      mass ratio
+    
+    spin1: spin of the primary black hole / Default: None
+           Not Implemented in most models
+    
```

Comment:
specify dimensionless spin where  -1 <= spin1 <= 1.

possibly even change the name to chi1, since spin1 is likely to be interpreted as dimensioned.

Reply:
Done!